examples/nft-nd/nft.go

```diff
@@ -247,14 +244,12 @@ func OnNEP17Payment(from interop.Hash160, amount int, data interface{}) {
 		tokIn = append(tokIn, std.Serialize(data)...)
 	}
 
-	tokenHash := crypto.Sha256(tokIn)
+	tokenHash := crypto.Ripemd160(tokIn)
```

Comment:
Why?

Reply:
Because of storage key-length restriction.

Reply:
But it's 64 bytes which should be enough for 32-byte hash plus some prefix?

Reply:
Even with base58 encoding.

Reply:
Base58 of 32 bytes can be 44 bytes and in some cases we also have prefix (1 byte) + holder (20).
Changing it this way was simpler and closer to C# implementation of NNS. Otherwise we will need to convert to/from base58 on every user boundary.

Reply:
Meh. Makes me wonder if 64 is a good limit (though it dates back to neo-project/neo#768 and has some reasoning behind it), not really critical though.